src/sentry/integrations/slack/webhooks/command.py

```diff
@@ -42,24 +36,16 @@ def is_team_linked_to_channel(organization: Organization, slack_request: SlackDM
     """Check if a Slack channel already has a team linked to it"""
     # Explicitly typing to satisfy mypy.
     is_linked: bool = ExternalActor.objects.filter(
-        organization=organization,
-        integration=slack_request.integration,
+        organization_id=organization,
```

Comment:
This technically is working now (django uses some meta to understand that the id column is a foreign key on this model) but won't work in the future that we actually break the FK.  Can we make this `organization_id=organization.id`?

Reply:
Must have missed this when making the change, good call!